src/lib.rs

```diff
@@ -43,10 +36,7 @@ impl Shaders {
     }
 
     fn get_program_code() -> (&'static str, &'static str) {
-        (
-            include_str!("imgui.vert"),
-            include_str!("imgui.frag"),
-        )
+        (include_str!("imgui.vert"), include_str!("imgui.frag"))
```

Comment:
Why did this get changed?

Reply:
Not exactly sure, I have my editor configured to run `rustfmt` on save, and a lot of stuff seemed to be reformatted as a result. Do you use custom settings for `rustfmt`? (I just use the defaults)